tools/topology/topology1/sof-smart-amplifier.m4

```diff
@@ -19,6 +19,11 @@ DEBUG_START
 # define the default macros.
 # define them in your specific platform .m4 if needed.
 
+#undefine the DYNAMIC flag (if enabled, save it) for smart amplifier as it uses volatile kcontrols
```

Comment:
@sathyap-chrome can you please fix my email ID to ranjani.sridharan@linux.intel.com

Reply:
sure